pkg/controller/plan/adapter/converter.go

```diff
@@ -230,42 +235,68 @@ func makeConversionContainer(pvc *v1.PersistentVolumeClaim, srcFormat, dstFormat
 	return container
 }
 
-func (c *Converter) ensureScratchPVC(sourcePVC *v1.PersistentVolumeClaim) (*v1.PersistentVolumeClaim, error) {
-	scratchPVC := &v1.PersistentVolumeClaim{}
-	err := c.Destination.Client.Get(context.Background(), client.ObjectKey{Name: getScratchPVCName(sourcePVC), Namespace: sourcePVC.Namespace}, scratchPVC)
+func (c *Converter) ensureScratchDV(sourcePVC *v1.PersistentVolumeClaim) (*cdi.DataVolume, error) {
+	scratchDV := &cdi.DataVolume{}
+	dvList := &cdi.DataVolumeList{}
+	label := client.MatchingLabels{planbase.AnnConversionSourcePVC: sourcePVC.Name}
+	err := c.Destination.Client.List(context.Background(), dvList, client.InNamespace(sourcePVC.Namespace), label)
 	if err != nil {
-		if k8serr.IsNotFound(err) {
-			scratchPVC := makeScratchPVC(sourcePVC)
-			c.Log.Info("Scratch pvc doesn't exist, creating...", "pvc", sourcePVC.Name)
-			err = c.Destination.Client.Create(context.Background(), scratchPVC, &client.CreateOptions{})
-		}
 		return nil, err
 	}
 
-	return scratchPVC, nil
-}
+	if len(dvList.Items) == 1 {
+		c.Log.Info("Found DV", "dv", dvList.Items[0].Name)
+		return &dvList.Items[0], nil
+	} else if len(dvList.Items) > 1 {
+		return nil, liberr.New("multiple scratch DVs found for pvc", "pvc", sourcePVC.Name)
+	}
+
+	// DV doesn't exist, create it
+	scratchDV = makeScratchDV(sourcePVC)
```

Comment:
```suggestion
	scratchDV := makeScratchDV(sourcePVC)
```
(and drop `scratchDV := &cdi.DataVolume{}` above)